lib/fastlane/plugin/clean_testflight_testers/actions/clean_testflight_testers_action.rb

```diff
@@ -45,12 +63,12 @@ def self.run(params)
         end
       end
 
-      def self.remove_tester(tester, app_id, dry_run)
+      def self.remove_tester(tester, app, dry_run)
```

Comment:
I hardly remember ids, so lets pass app object and show app name when printing and use apple_id when deleting.